gtsam/nonlinear/nonlinear.i

```diff
@@ -572,8 +572,8 @@ virtual class GncOptimizer {
   gtsam::Values optimize();
 };
 
-typedef gtsam::GncOptimizer<gtsam::GncParams<gtsam::GaussNewtonParams> > GncGaussNewtonOptimizer;
-typedef gtsam::GncOptimizer<gtsam::GncParams<gtsam::LevenbergMarquardtParams> > GncLMOptimizer;
+typedef gtsam::GncOptimizer<gtsam::GncParams<gtsam::GaussNewtonParams>> GncGaussNewtonOptimizer;
```

Comment:
this sort of syntax used to throw parser errors back in the day, since >> was an operator itself.
https://stackoverflow.com/questions/6695261/template-within-template-why-should-be-within-a-nested-template-arg

Is that fixed in C++11 ?

Reply:
C++ does not parse .i files…